Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,184 @@
+          pradeo.device.declaredOperatingSystemVersion: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.declaredOperatingSystemVersion}}'
+          pradeo.device.declaredOperatingSystemSecurityPatchDate: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.declaredOperatingSystemSecurityPatchDate}}'
+          pradeo.device.declaredModel: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.declaredModel}}'
+          pradeo.device.lastConnection: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.enrollmentStatus.lastConnection}}'
+          pradeo.device.coupled: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.enrollmentStatus.coupled}}'
+          pradeo.device.mdmId: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.emmDeviceInfo.externalId}}'
+          pradeo.device.emm: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.emmDeviceInfo.emm}}'
+          pradeo.compliance.matchedResponseRules: '{{json_event.message.content.deviceApplication.compliance.matchedResponseRules}}'
+          pradeo.application.id: '{{json_event.message.content.deviceApplicationCompliance.application.id}}'
+          pradeo.application.package: '{{json_event.message.content.deviceApplicationCompliance.application.package.package}}'
+          pradeo.application.system: '{{json_event.message.content.deviceApplicationCompliance.application.package.system}}'
+          pradeo.application.version: '{{json_event.message.content.deviceApplicationCompliance.application.version}}'
+          pradeo.application.versionCode: '{{json_event.message.content.deviceApplicationCompliance.application.versionCode}}'
+          pradeo.application.name: '{{json_event.message.content.deviceApplicationCompliance.application.name}}'
+          pradeo.application.md5: '{{json_event.message.content.deviceApplicationCompliance.application.md5}}'
+          pradeo.application.sha1: '{{json_event.message.content.deviceApplicationCompliance.application.sha1}}'
+          pradeo.application.sha256: '{{json_event.message.content.deviceApplicationCompliance.application.sha256}}'
+          pradeo.detection.status: '{{json_event.message.content.deviceApplicationCompliance.status}}'
+      - filter: '{{json_event.message.type == "DeviceComplianceUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.deviceCompliance.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.deviceCompliance.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.deviceCompliance.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.deviceCompliance.device.email}}'
```

[`user.email`](https://www.elastic.co/guide/en/ecs/8.11/ecs-user.html#field-user-email) is more suitable.
```suggestion
          user.email: '{{json_event.message.content.deviceCompliance.device.email}}'
```